core/src/akademia/medievilai/GUIParams.java

```diff
@@ -0,0 +1,8 @@
+package akademia.medievilai;
+
+public class GUIParams {
+    final static int SCREEN_WIDTH = 800;
+    final static int SCREEN_HEIGHT = 600;
+    final static int CARDVIEW_WIDTH = 100;
```

Comment:
CardView to dwa słowa rozróżnione wielką literą, w tej konwencji dajemy _ w każdej takiej sytuacji więc powinno być CARD_VIEW_WIDTH